.gitignore

```diff
@@ -23,4 +23,16 @@ test/compiled/*
 .idea/
 celerybeat-schedule
 deployments.json
-.DS_Store
\ No newline at end of file
+.DS_Store
+jsapp/fonts/*.eot
+jsapp/fonts/*.svg
+jsapp/fonts/*.ttf
+jsapp/fonts/*.woff
+jsapp/fonts/*.woff2
+jsapp/fonts/*.scss
+jsapp/fonts/*.css
+jsapp/fonts/*.md
+jsapp/fonts/*.ijmap
+jsapp/fonts/*.otf
+jsapp/fonts/codepoints
+jsapp/fonts/*.html
```

Comment:
Make sure you delete `jsapp/fonts/.gitignore` :)

Reply:
Give me the time to do it ;-)